<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>622</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(139, 173, 227);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>201</width>
      <height>107</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="rb_enter">
       <property name="text">
        <string>Войти</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_rem">
       <property name="text">
        <string>Запомнить меня</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_register">
       <property name="text">
        <string>Регистрация</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="login">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>197</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: black;
padding: 1px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="passw1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>197</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: black;
padding: 1px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="passw2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>197</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: black;
padding: 1px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_log">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Введите ваш логин</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_pw">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите ваш пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_pw2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Подтвердите ваш пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="enter_denied">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Неверный логин или пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>15</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 235, 151);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: black;
padding: 1px;
</string>
    </property>
    <property name="text">
     <string>Играть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>Вы успешно зарегистрированы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Поля не могут быть пустыми</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Пароли не совпадают</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_changing">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 0, 255);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: black;
padding: 1px;
</string>
    </property>
    <property name="text">
     <string>Смена пароля</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_inform">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>510</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
background-color: rgb(0, 255, 255);
border-width: 1px;
border-radius: 5px;
border-color: black;
padding: 1px;
</string>
    </property>
    <property name="text">
     <string>Информация</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tabl">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>540</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
background-color: rgb(165, 255, 176);
border-width: 1px;
border-radius: 5px;
border-color: black;
padding: 1px;
</string>
    </property>
    <property name="text">
     <string>Таблица рекордов</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>319</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
